### PR TITLE
Update documentation on shoebox.

### PR DESCRIPTION
**Pull request by burritoIand**
changes:
  - Update README to add description of alternative solution for fastboot's shoebox behavior using the Application Adapter.

https://github.com/ember-fastboot/ember-cli-fastboot/issues/761

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -414,7 +414,51 @@ export default Ember.Route.extend({
...
+    }
+
+    let result = this.fastboot.shoebox.retrieve(key);
+
+    if (!result) {
+      result = await super.findRecord(...arguments);
+    }
+
+     // must deep-copy for clean serialization.
+     return JSON.parse(JSON.stringify(result));
```
Comment (rwjblue) [inline README.md:450]:
```suggestion
    // must deep-copy for clean serialization.
    return JSON.parse(JSON.stringify(result));
```

File: README.md [hunk 2]
```diff
@@ -414,7 +414,51 @@ export default Ember.Route.extend({
 
 Shoebox gives you great capabilities, but using it in the real app is pretty rough. Have you ever thought that such kind of logic should be done behind the scenes? In a large codebase, defining `fastboot.isFastBoot` conditionals can be a daunting task. Furthermore, it generates a lot of boilerplate code, which obscures the solution. Sooner or later coupling with `shoebox` will spread over all routes.
 
-Fortunately, there is an addon called [ember-data-storefront](https://embermap.github.io/ember-data-storefront/) that can help to alleviate this pain, thanks to its Fastboot mixin: https://embermap.github.io/ember-data-storefront/docs/guides/fastboot.
+#### Solution: Application Adapter
+
+One way to abstract the shoebox data storage mechanics is to move the logic into
+the Application Adapter as shown below.
+```
```
Comment (rwjblue) [inline README.md:421]:
```suggestion
the Application Adapter as shown below.

```js
```

File: README.md [hunk 3]
```diff
@@ -414,7 +414,51 @@ export default Ember.Route.extend({
...
+export default class ApplicationAdapter extends JSONAPIAdapter.extend(
+  ...
```
Comment (rwjblue) [inline README.md:423]:
```suggestion
  // ...snip...
```

**Comment by burritoIand** [comment]
@rwjblue updated per feedback

**Comment by kratiahuja** [comment]
LGTM! Thanks for doing this. 

@rwjblue does this look good to you?

**Pull request merged**